hackathon_site/dashboard/frontend/src/components/teamDetail/TeamInfoTable/TeamInfoTable.tsx

```diff
@@ -52,7 +56,16 @@ export const TeamInfoTable = () => {
                                         <Checkbox
                                             checked={row.id_provided}
                                             color="primary"
-                                            data-testid="checkbox"
+                                            disabled={isParticipantIdLoading}
+                                            data-testid={`id-provided-check-${row.id}`}
```

Comment:
what is this data-testid used for? I am assuming it is used only for the jest test, is that correct?

Reply:
yes that is correct